lib/Extension/Document/Areabrick/AreabrickManagerInterface.php

```diff
@@ -58,35 +58,4 @@ public function getBricks(): array;
      */
     public function getBrickIds(): array;
 
-    /**
-     * Enables an areabrick
-     *
-     * @param string $id
-     */
-    public function enable(string $id);
-
-    /**
-     * Disables an areabrick
-     *
-     * @param string $id
-     */
-    public function disable(string $id);
-
-    /**
-     * Determines if an areabrick is enabled. Bricks are enabled by default an can be switched off by setting
-     * the state explicitely to false in the extension config.
-     *
-     * @param string $id
-     *
-     * @return bool
-     */
-    public function isEnabled(string $id): bool;
-
-    /**
-     * Enables/disables an areabrick
-     *
-     * @param string $id
-     * @param bool $state
-     */
-    public function setState(string $id, bool $state);
```

Comment:
Should we add the `@deprecated` annotations to the interface in `10.5.x`, too? Currently, it's only in the concrete implementation.